### PR TITLE
exclude synonym on 'acquired polycythemia'

### LINKED ISSUE #7178: MONDO:0023193 & MONDO:0016031[Merge]
**Mondo term (ID and Label)**
MONDO:0023193 Friedman Goodman syndrome

**Reason for deprecation**
Per Orphanet [1969](http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=1969) these are the same condition.

**Term to be merged with**
MONDO:0016031  facial dysmorphism-anorexia-cachexia-eye and skin anomalies syndrome 
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close https://github.com/monarch-initiative/mondo/issues/7176

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -40055,7 +40055,7 @@ subset: nord_rare {source="MONDO:NORD"}
 subset: rare
 synonym: "acquired polycythemia" EXACT [MONDO:patterns/acquired]
 synonym: "acquired polycythemia (disease)" EXACT [MONDO:patterns/acquired]
-synonym: "polycythemia, secondary" RELATED [DOID:2834]
+synonym: "polycythemia, secondary" RELATED EXCLUDE [DOID:2834]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:40058]: I don't follow the trail from the original ticket to merge two terms to adding `EXCLUDE` as a [synonym type](https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#synonym-types)

**Review by twhetzel — COMMENTED** [review]
comment inline

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:40058]: Ha - I linked the wrong ticket, so that is why it did not make sense! I fixed it now. Thanks!

**Review by twhetzel — APPROVED** [review]
LGTM

**Pull request merged**